IpmiFeaturePkg/GenericIpmi/StandaloneMm/StandaloneMmGenericIpmi.c

```diff
@@ -0,0 +1,115 @@
+/** @file
+  Generic Standalone MM IPMI stack driver
+
+  @copyright
+  Copyright 1999 - 2021 Intel Corporation. <BR>
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+//
+// Statements that include other files
+//
+#include <Uefi.h>

```

Comment:
Can you double check these includes? TimerLib doesn't appear to be needed

Reply:
Remove few non-required header files.